signup-private-modules.js

```diff
@@ -10,7 +10,7 @@ require('./lib/sharedNemo').then(function(nemo) {
     bail: true
   }, function(t) {
     return P.all([
-      nemo.driver.get(urlOf('/settings/billing')),
+      nemo.driver.get(urlOf('/settings/billing#payment-information')),
```

Comment:
Okey doke. Weirds me out to see in a test, since I'm used to HTTP tests. Makes sense though.